### PR TITLE
Add lipid trait terms for GWAS/EFO

### LINKED ISSUE #257: Lipid trait terms for GWAS/EFO
- [x] Compile a table of candidate terms that includes standard identifiers for component ontology terms
    - [x] EFO [issue #1882](https://github.com/EBISPOT/efo/issues/1882)
    - [x] EFO [issue 1905#](https://github.com/EBISPOT/efo/issues/1905#issuecomment-1568220743)
    - [x] EFO [issue #1924](https://github.com/EBISPOT/efo/issues/1924#issuecomment-1571972259)
- [x] Fill in the appropriate DOS-DP template table and create new terms

### PR DESCRIPTION
**Pull request by rays22**
Update some lipid trait term labels.
Add new level of lipid in blood serum trait terms.

If applied, this is part of a series to fix #257.

Addresses https://github.com/EBISPOT/efo/issues/1905 
Addresses https://github.com/EBISPOT/efo/issues/1924

### CODE REVIEW

**Comment by rays22** [comment]
This error:


Level | Rule Name | Subject | Property | Value
-- | -- | -- | -- | --
ERROR | biological-attribute-child-violation | OBA:0003476 | rdfs:subClassOf | Not a subclass of biological attribute


is cause by the recent obsoletion of [GO:0070265	necrotic cell death](https://www.ebi.ac.uk/QuickGO/term/GO:0070265)

The rest of the errors can be fixed by refreshing chebi-slim and the OBA imports.

**Comment by matentzn** [comment]
Move the cell death one into oba-edit.obo the way we did with some of the other ones?

File: src/ontology/oba-edit.obo [hunk 1]
```diff
@@ -1324,6 +1324,15 @@ is_a: OBA:0000001 ! biological attribute
...
 is_a: OBA:0000001 ! biological attribute
 
+[Term]
+id: OBA:0003476
+name: necrotic cell death amount
+def: "The amount of a necrotic cell death." []
+synonym: "amount of necrotic cell death" EXACT []
+is_a: OBA:0100002 ! biological process attribute
+property_value: http://purl.org/dc/elements/1.1/date "2023-06-16T20:21:00Z" xsd:dateTime
+property_value: http://www.geneontology.org/formats/oboInOwl#http://purl.org/dc/terms/contributor https://orcid.org/0000-0001-8314-2140
```
Comment (matentzn) [inline src/ontology/oba-edit.obo:1334]:
```suggestion
property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0001-8314-2140
```

File: src/patterns/definitions.owl [hunk 2]
```diff
@@ -63759,8 +63803,8 @@ EquivalentClasses(<http://purl.obolibrary.org/obo/OBA_2040789> ObjectIntersectio
...
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2040790> "blood serum dehydrogenase/reductase SDR family member 6 amount"^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/OBA_2040790> "The amount of a 3-hydroxybutyrate dehydrogenase type 2 when measured in blood serum."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2040790> "blood serum 3-hydroxybutyrate dehydrogenase type 2 amount"^^xsd:string)
```
Comment (matentzn) [inline src/patterns/definitions.owl:63807]: The label change is pretty significant - is this intended?

File: src/patterns/definitions.owl [hunk 3]
```diff
@@ -69394,8 +69438,8 @@ EquivalentClasses(<http://purl.obolibrary.org/obo/OBA_2041595> ObjectIntersectio
...
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2041596> "blood serum flavin-containing monooxygenase 3 amount"^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/OBA_2041596> "The amount of a dimethylaniline monooxygenase [N-oxide-forming] 3 when measured in blood serum."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2041596> "blood serum dimethylaniline monooxygenase [N-oxide-forming] 3 amount"^^xsd:string)
```
Comment (matentzn) [inline src/patterns/definitions.owl:69442]: Seems like a bigger label change - is this intended?

**Review by matentzn — COMMENTED** [review]
One question, one change request

File: src/patterns/definitions.owl (hunk 2 above)
Comment (rays22) [reply src/patterns/definitions.owl:63807]: Good catch! I think I reused an ID that I had already assigned in another PR.

Comment (rays22) [reply src/patterns/definitions.owl:63807]: I will double check the OBA IDs.

File: src/patterns/definitions.owl (hunk 3 above)
Comment (rays22) [reply src/patterns/definitions.owl:69442]: Again, one of the automatic pipelines replaced the term label and the PR name in the text definition with [one of the synonyms of PR:000007576](https://www.ebi.ac.uk/ols4/ontologies/pr/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPR_000007576).
My data table entry has not changed:
```
defined_class	defined_class_name	entity	entity_name	attribute	attribute_name	location	location_name	exact_synonyms	xrefs
OBA:2041596	level of flavin-containing monooxygenase 3 in blood serum	PR:000007576	flavin-containing monooxygenase 3	PATO:0000070	amount	UBERON:0001977	blood serum
```

**Review by matentzn — APPROVED** [review]

**Pull request merged**